src/OrchardCore/OrchardCore.ResourceManagement/OrchardCore.ResourceManagement.csproj

```diff
@@ -1,10 +1,15 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Caching.Abstractions" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\OrchardCore.Abstractions\OrchardCore.Abstractions.csproj" />
```

Comment:
removed

Reply:
Ideal, should be able to clean up all the `System.Environment.NewLine` in `ResourceManager` I had to add to avoid namespace conflict with `Orchard.Environment`

Reply:
Okay, done on my side.